src/main/java/hudson/plugins/emailext/plugins/recipients/CulpritsRecipientProvider.java

```diff
@@ -34,16 +36,18 @@ public CulpritsRecipientProvider() {
     @Override
     public void addRecipients(ExtendedEmailPublisherContext context, EnvVars env, Set<InternetAddress> to, Set<InternetAddress> cc, Set<InternetAddress> bcc) {
         ExtendedEmailPublisherDescriptor descriptor = Jenkins.getInstance().getDescriptorByType(ExtendedEmailPublisherDescriptor.class);
-        Set<User> users = context.getBuild().getCulprits();
-
-        for (User user : users) {
-            if (!EmailRecipientUtils.isExcludedRecipient(user, context.getListener())) {
-                String userAddress = EmailRecipientUtils.getUserConfiguredEmail(user);
-                if (userAddress != null) {
-                    descriptor.debug(context.getListener().getLogger(), "Adding user address %s, they were not considered an excluded committer", userAddress);
-                    EmailRecipientUtils.addAddressesFromRecipientList(to, cc, bcc, userAddress, env, context.getListener());
-                } else {
-                    context.getListener().getLogger().println("Failed to send e-mail to " + user.getFullName() + " because no e-mail address is known, and no default e-mail domain is configured");
+        Run<?,?> run = context.getRun();
+        if(run instanceof AbstractBuild) {
+            Set<User> users = ((AbstractBuild<?,?>)run).getCulprits();
+            for (User user : users) {
+                if (!EmailRecipientUtils.isExcludedRecipient(user, context.getListener())) {
+                    String userAddress = EmailRecipientUtils.getUserConfiguredEmail(user);
+                    if (userAddress != null) {
+                        descriptor.debug(context.getListener().getLogger(), "Adding user address %s, they were not considered an excluded committer", userAddress);
+                        EmailRecipientUtils.addAddressesFromRecipientList(to, cc, bcc, userAddress, env, context.getListener());
+                    } else {
+                        context.getListener().getLogger().println("Failed to send e-mail to " + user.getFullName() + " because no e-mail address is known, and no default e-mail domain is configured");
+                    }
```

Comment:
Correct, currently I don't know of a way to get culprits when run is a WorkflowRun.